src/bundle.c

```diff
@@ -171,6 +171,33 @@ static int unload_tracked_bundle(const char *bundle_name)
 	return EBUNDLE_NOT_TRACKED;
 }
 
+/* Check if bundle is included by any subscribed bundles */
+bool is_included(const char *bundle_name, struct manifest *mom)
+{
+	bool ret = false;
+	struct list *b, *i;
+
+	b = list_head(mom->submanifests);
+	while (b) {
+                struct manifest *bundle = b->data;
+		b = b->next;
+
+                i = list_head(bundle->includes);
+                while (i) {
+                        char *name = i->data;
+                        i = i->next;
+
+                        if (strcmp(name, bundle_name) == 0) {
```

Comment:
Can we report the "name" here, so that users will know and decide if they want to attempt removing it as well?